stdlib/public/SDK/CoreGraphics/CGFloat.swift.gyb

```diff
@@ -28,7 +28,7 @@ public struct CGFloat {
   /// The native type used to store the CGFloat, which is Float on
   /// 32-bit architectures and Double on 64-bit architectures.
   public typealias NativeType = Float
-#elseif arch(x86_64) || arch(arm64)
+#elseif arch(x86_64) || arch(arm64) || arch(s390x)
```

Comment:
This overlay is specific to Apple platforms, so this change shouldn't be needed.